Doctor-Appointment-App/l_static/assets/plugins/bootstrap-tagsinput/js/bootstrap-tagsinput.js

```diff
@@ -0,0 +1,682 @@
+/*
+ * bootstrap-tagsinput v0.8.0
+ * 
+ */
+
+(function ($) {
+  "use strict";
+
+  var defaultOptions = {
+    tagClass: function(item) {
+      return 'badge badge-info';
+    },
+    focusClass: 'focus',
+    itemValue: function(item) {
+      return item ? item.toString() : item;
+    },
+    itemText: function(item) {
+      return this.itemValue(item);
+    },
+    itemTitle: function(item) {
+      return null;
+    },
+    freeInput: true,
+    addOnBlur: true,
+    maxTags: undefined,
+    maxChars: undefined,
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/BuildForSDGCohort2/Team-317-Doctor-Appointment/pullRequest?prid=6207082)